pkg/kubelet/types.go

```diff
@@ -68,12 +69,19 @@ type PodUpdate struct {
 }
 
 // GetPodFullName returns a name that uniquely identifies a pod across all config sources.
+// NOTE: If changed ParsePodFullName must be also updated.
 func GetPodFullName(pod *api.Pod) string {
 	// Use underscore as the delimiter because it is not allowed in pod name
 	// (DNS subdomain format), while allowed in the container name format.
 	return fmt.Sprintf("%s_%s", pod.Name, pod.Namespace)
 }
 
+// ParsePodFullName parses full name generated by GetPodFullName and returns parts of it.
```

Comment:
I've already added a ParsePodFullName function in My PR (#5401). You'll need to remove this one.